src/Orleans.Membership.ServiceFabric/Orleans.Membership.ServiceFabric.csproj

```diff
@@ -1,15 +1,15 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <PackageId>Microsoft.Orleans.ServiceFabric</PackageId>
```

Comment:
We should probably create the `Microsoft.Orleans.ServiceFabric` meta-package. Do we do it now or in a different PR? I suppose it's better to do it in the same PR where we split the package

Reply:
Oh originally I had the metapackage, but I figured it was just noise. I can add it again.

Reply:
Yeah, I think we should do that